casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -105,14 +105,21 @@ object BlockAPI {
       _.withCasper[ApiErr[String]](
         casper =>
           for {
-            resultOpt <- triggerProposeF(casper, isAsync)
-            // wait till propose pipe assign proposeID to propose and resolve proposeIDDef
-            r <- resultOpt match {
-                  case Some(Right(blockHash)) =>
-                    logSucess(s"Success! Block ${blockHash.base16String} created and added.")
-                  case Some(Left(seqNum)) =>
-                    logSucess(s"Success: proposing block with seqNum ${seqNum}")
-                  case None => logDebug(s"Failure: another propose is in progress")
+            // Trigger propose
+            proposerResult <- triggerProposeF(casper, isAsync)
+            r <- proposerResult match {
+                  case ProposerEmpty =>
+                    logDebug(s"Failure: another propose is in progress")
```

Comment:
@zsluedem I fixed the message to stay in the same format. Can you point out where else we are depending on the returning text except in pyrchain?

Reply:
some of the integration tests depend on the logs. But the tests are passing. I think it is fine.